app/src/main/java/com/example/mykotlincalculator21/utils/StringUtils.kt

```diff
@@ -1,8 +1,9 @@
 package com.example.mykotlincalculator21.utils
 
-const val EMPTY_STRING = ""
-const val OPERATOR_SUM = "+"
-const val OPERATOR_MINUS = "-"
-const val OPERATOR_DIVIDE = "/"
-const val OPERATOR_MULTIPLY = "*"
-const val ERROR_MESSAGE = "Error, unknown Symbol used"
+object StringUtils {
+     val EMPTY_STRING = ""
+     val OPERATOR_SUM = "+"
+     val OPERATOR_MINUS = "-"
+     val OPERATOR_DIVIDE = "/"
+     val OPERATOR_MULTIPLY = "*"
+}
```

Comment:
why do you put all the constants inside a single object?

Reply:
Because whenever i try to use "class" i can not import the values that i need